### PR TITLE
Hotfix handle turn

### PR DESCRIPTION
**Pull request by J0hanna01**

### CODE REVIEW

File: src/main/java/se/group3/backend/domain/cards/ActionCard.java [hunk 1]
```diff
@@ -40,16 +45,32 @@ public int getMoneyAmount() {
...
 
-    @Override
+/*    @Override
     public String toString() {
         return "ActionCard{" +
                 "id='" + getId() + '\'' +
                 "name='" + getName() + '\'' +
                 '}';
-    }
+    }*/
```
Comment (thomas-schleicher) [inline src/main/java/se/group3/backend/domain/cards/ActionCard.java]: remove / fix commented code

File: src/main/java/se/group3/backend/domain/cards/Card.java [hunk 2]
```diff
@@ -21,7 +21,7 @@ public String getId() {
         return id;
     }
     // Abstract method to perform the card's action
-    public abstract void performAction(Player player);
+   // public abstract void performAction(Player player);*/
```
Comment (thomas-schleicher) [inline src/main/java/se/group3/backend/domain/cards/Card.java:24]: same as above

File: src/main/java/se/group3/backend/domain/cards/CareerCard.java [hunk 3]
```diff
@@ -45,7 +64,7 @@ public String toString() {
                 ", bonus=" + bonus +
                 ", needsDiploma=" + needsDiploma +
                 '}';
-    }
+    }*/
```
Comment (thomas-schleicher) [inline src/main/java/se/group3/backend/domain/cards/CareerCard.java:67]: same as above

File: src/main/java/se/group3/backend/domain/cards/HouseCard.java [hunk 4]
```diff
@@ -38,5 +61,5 @@ public String toString() {
                 "id='" + getId() + '\'' +
                 "name='" + getName() + '\'' +
                 '}';
-    }
+    }*/
```
Comment (thomas-schleicher) [inline src/main/java/se/group3/backend/domain/cards/HouseCard.java:64]: same as above

**Review by Haxe33 — APPROVED** [review]

File: src/main/java/se/group3/backend/domain/cards/ActionCard.java (hunk 1 above)
Comment (J0hanna01) [reply src/main/java/se/group3/backend/domain/cards/ActionCard.java]: done

**Review by thomas-schleicher — APPROVED** [review]

**Pull request merged**